### PR TITLE
Add more params for inkwell

### PR DESCRIPTION
**Pull request by ZhuJHua**

### CODE REVIEW

File: lib/src/rive_animated_icon.dart [hunk 1]
```diff
@@ -51,8 +53,22 @@ class RiveAnimatedIcon extends StatefulWidget {
...
+  final MouseCursor mouseCursor;
+
+  /// [hoverColor] is for rendering the hover color when the icon is hovered
+  /// Default value: [Colors.transparent]
+  /// it's an  optional parameter for [RiveAnimatedIcon]
+  final Color hoverColor;
+
   /// [strokeWidth] is for rendering the animated icon with respected stroke width
-  /// Default value: [2]
+  /// Default value: [InkSplash.splashFactory]
```
Comment (nikkieke) [inline lib/src/rive_animated_icon.dart]: There seems to be a typo here. Default value for stroke width is 2.

**Comment by nikkieke** [comment]
Hi @ZhuJHua 
Thanks for taking the time to contribute. There seems to be a slight issue on Inkwell and Rive assets. On Flutter web hover color doesn't show when you hover on the icons however, splash color shows when you hover on the icons. You can test this out to see what I mean. 

I am still looking into what could be the cause of this. In the meantime, can you remove the hover color param and fix the typo. Once you make these adjustments I'd be happy to merge as everything else LGTM.

**Comment by ZhuJHua** [comment]
> Hi @ZhuJHua Thanks for taking the time to contribute. There seems to be a slight issue on Inkwell and Rive assets. On Flutter web hover color doesn't show when you hover on the icons however, splash color shows when you hover on the icons. You can test this out to see what I mean.
> 
> I am still looking into what could be the cause of this. In the meantime, can you remove the hover color param and fix the typo. Once you make these adjustments I'd be happy to merge as everything else LGTM.

Thank you for your response. I'm actually doing this because. Sometimes I don't need a `InkWell` , I just need an infinite loop icon,  such as used instead of `CircularProgressIndicator`,  so I prefer to add a new method, simply return a icon instead of buttons.

**Comment by nikkieke** [comment]
> Thank you for your response. I'm actually doing this because. Sometimes I don't need a `InkWell` , I just need an infinite loop icon, such as used instead of `CircularProgressIndicator`, so I prefer to add a new method, simply return a icon instead of buttons.

Okay, that sounds great.👍 You can add a new method that allows someone to choose between returning an icon or a botton.
Can you make a separate PR for this.

**Comment by ZhuJHua** [comment]
> Can you make a separate PR for this.

Of course

**Review by nikkieke — APPROVED** [review]
Everything looks good! Thank you!

**Pull request merged**